### DISCUSSION TITLE
Preciso criar um menu multi-level no vuetify

### DISCUSSION THREAD

**Discussion by DinnoFe** [frontendbr/forum#1674] [Perguntas e Respostas]
Ola pessoas tudo bem, então to trabalhando num projeto e to preso a um dia na criação de um menu drawer, oque eu preciso dele, que ele crie um menu lateral que se sobrepõe na tela ok ate ae e tranquilo, agora oque complica e que preciso que os submenus dele chamem outros drawers que se sobrepõe ao pai, mas nao to conseguindo fazer isso, alguém tem uma luz para dar ou conheça um componente ja pronto ?, estou usando vue/vuetify

**Comment by adsonvinicius** [comment]
Quando precisei e não tinha precisei fazer um: https://github.com/adsonvinicius/v-menu-multi-level
